script/BaseScript.s.sol

```diff
@@ -353,36 +320,24 @@ contract BaseScript is BaseData, CREATE3Script, Utils {
         baseInput.rateLimitConfig.window = vm.parseJsonUint(json, ".rateLimitConfig.window");
     }
 
-    function createL1YnOFTAdapterUpgradeableProxySalt() internal view returns (bytes32 _salt) {
-        _salt = createSalt("L1YnOFTAdapterProxy");
-    }
-
-    function createL1YnOFTAdapterUpgradeableSalt() internal view returns (bytes32 _salt) {
-        _salt = createSalt("L1YnOFTAdapter");
+    function createOFTAdapterProxySalt() internal view returns (bytes32 _salt) {
```

Comment:
Simplifies function names, consolidates salts to have the same salt for oft adapters for L1 and L2. Earlier we were using different salts which was definitely a mistake on our part. Going forward we can have the same oftAdapter and timelock addresses for all networks (L1 or L2)

Reply:
this pull request is closed, i told you not to go down this path